docs/modules/reference/pages/glossary.adoc

```diff
@@ -109,31 +115,34 @@ It provides visualization capabilities for data indexed in an Elasticsearch clus
 xref:operation:admin/webui/opsboard/dashlet/ksc.adoc#ksc[KSC reports]:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data.
 They let you display data from different devices and sources (SNMP, ICMP, HTTP) on one page.
 
-Kubernetes:: An open-source container orchestration system for automating software deployments, scaling, and management.
+https://kubernetes.io/[Kubernetes]:: An open-source container orchestration system for automating software deployments, scaling, and management.
 Originally designed by Google, the Cloud Native Computing Foundation now maintains it.
-(See https://kubernetes.io/[Kubernetes].)
 
 Location:: Also referred to as "monitoring location" or "Minion location".
 Nodes assigned to a monitoring location will be monitored by Minions assigned to the same location.
 This is distinct from geolocation, which is the actual physical location of the node.
 
+Management Information Base (MIB):: A database used for managing entities on a network.
+Used in {page-component-title} to facilitate Transparent Bridge Discovery.
+(See xref:operation:topology/enlinkd/layer-2/bridge-discovery.adoc[Bridge Discovery].)
```

Comment:
MIBs are text files that describe SNMP data.  There is a tool to compile them into XML config files, though there isn't any documentation on that tool.
```suggestion
Management Information Base (MIB):: A file provided by a hardware or software vendor that describes SNMP objects provided by their products.
The MIB files can be processed to convert the definition into data collection and event objects for {page-component-title} to work with.
Many of the major vendor MIB definitions are included out of the box.
```